CHANGES.rdoc

```diff
@@ -4,6 +4,10 @@ This document describes the relevant changes between releases of the
 _hawkular-client_ project.
 
 
+=== v 3.0.3
+* Standardized Exceptions under the Hawkular namespace, and the old names were deprecated, here is the list:
+1. <code>Hawkular::BaseClient::HawkularException</code> -> <code>Hawkular::Exception</code>
+2. <code>Hawkular::BaseClient::HawkularConnectionException</code> -> <code>Hawkular::ConnectionException</code>
```

Comment:
I don't think it is a breaking change because the API does not change, as the old exceptions are still available. Maybe a minor version bump instead of a patch one?